lib/buyers/tests/test_api.py

```diff
@@ -374,3 +374,22 @@ def test_bad_uuid(self):
     def test_empty_post(self):
         res = self.client.post(self.list_url, data={})
         eq_(res.status_code, 422)
+
+
+class TestBuyerClose(APITest):
+
+    def setUp(self):
+        self.uuid = 'sample:uid'
+        self.buyer = Buyer.objects.create(uuid=self.uuid)
+        self.url = reverse('generic:buyer-close', kwargs={'pk': self.buyer.pk})
+
+    def test_close(self):
+        res = self.client.post(self.url, data={'uuid': self.uuid})
+        eq_(res.status_code, 204)
+        eq_(self.buyer.reget().active, False, res.content)
```

Comment:
that's in the tests for the close() method: https://github.com/mozilla/solitude/commit/2fd58fedf5fb3af517141248acca43f4ad8bdca7#diff-ba0f8932616d767f6d87927a25578a6eR120